groups/tooling/README.md

```diff
@@ -0,0 +1,97 @@
+# Tooling Group Initiatives
+
+> This "living document" outlines the initiatives that have come out of User Feedback Sessions for the Tooling group.
+
+<!-- TOC depthFrom:2 -->
+
+- [Add Common Filesystem Operations to Core](#add-common-filesystem-operations-to-core)
+  - [Summary](#summary)
+  - [Motivation](#motivation)
+  - [Example 1: The Bundler](#example-1-the-bundler)
+  - [Example 2: The Test Case](#example-2-the-test-case)
+  - [Notes](#notes)
+
+<!-- /TOC -->
+
+## Add Common Filesystem Operations to Core
+
+### Summary
+
+Two (2) filesystem operations should be added to Node.js Core:
```

Comment:
big +1, its not uncommon to have 15-25+ `mkdirp` modules in node_modules, and about the same for `fs-extra`